pkg/cloud/amazon/common.go

```diff
@@ -61,6 +67,26 @@ func ResolveRegionWithoutOptions() (string, error) {
 	return ResolveRegion("", "")
 }
 
+func RememberRegion(kubeClient kubernetes.Interface, namespace string, region string) error {
```

Comment:
I would prefer a more general solution. Please could you move this functions into `pkg/kub/jx-install-config.go` and rename the `AWS_REGION` to `Region`, that it can be reused with other cloud providers such as GKE.

Reply:
Good point. Done :)